.buildbot.sh

```diff
@@ -35,6 +35,15 @@ cargo fmt --all -- --check
 rustup toolchain install stable
 rustup default stable
 
+# Later on we are going to need to install cargo-deny and mdbook. We kick the
+# install jobs off now so that at least some work (e.g. downloading crates) can
+# happen in parallel, speeding up the overall process.
+
+cargo_deny_mdbook_tmp=$(mktemp)
+( cargo install --locked cargo-deny ; cargo install --locked mdbook ) \
```

Comment:
I see there are also some more `cargo install` calls happening (for I believe decently sized wasm packages).
I'm not sure if they benefit from this parallelism too, perhaps another patch?.